packages/plugin/src/theia.d.ts

```diff
@@ -81,6 +81,11 @@ declare module '@theia/plugin' {
          */
         readonly pluginPath: string;
 
+        /**
+         * The uri of the directory containing the this plugin-in.
```

Comment:
```suggestion
         * The uri of the directory containing this plug-in.
```

Reply:
There were 2 typos, only one is fixed
Still remain `plugin-in` typo

Reply:
Come-on, "plugin-in" is a good word :-)

Reply:
yep it's 'avant-garde'